### PR TITLE
upgrade to Fabric8.io Kubernetes client 4.10.2

### PR DESCRIPTION
**Pull request by FWiesner**
Attempt to get the client compatible with fabric8 4.10.2 to solve https://github.com/snowdrop/istio-java-api/issues/94

### CODE REVIEW

**Comment by rohanKanojia** [comment]
@iocanel @metacosm @geoand: Could you please take a look into this whenever you get some time?

File: istio-model/src/main/java/me/snowdrop/istio/api/networking/v1beta1/ServerTLSSettingsProtocol.java [hunk 1]
```diff
@@ -28,26 +30,31 @@
     /**
      * Automatically choose the optimal TLS version.
      */
+    @JsonProperty("TLS_AUTO")
     Server_TLSOptions_TLS_AUTO(0),
 
     /**
      * TLS version 1.0
      */
+    @JsonProperty("TLSV1_0")
```
Comment (metacosm) [inline istio-model/src/main/java/me/snowdrop/istio/api/networking/v1beta1/ServerTLSSettingsProtocol.java:39]: Do you have an example that we could check in that exhibits the serialization issue requiring this change?

Comment (FWiesner) [reply istio-model/src/main/java/me/snowdrop/istio/api/networking/v1beta1/ServerTLSSettingsProtocol.java:39]: Sure @metacosm - I'm not sure whether the document is functionally sane (combining min TLS version with PASSTHROUGH), but you will see the de-serialization issue with this document

```yaml
apiVersion: networking.istio.io/v1beta1
kind: Gateway
metadata:
  name: my-egress-gw
  namespace: istio-trials
spec:
  selector:
    istio: egressgateway
  servers:
    - hosts:
        - foobar.com
      port:
        number: 80
        name: http-0
        protocol: HTTP
    - hosts:
        - foobar.com
      port:
        number: 443
        name: tls-0
        protocol: TLS
      tls:
        mode: PASSTHROUGH
        minProtocolVersion: TLSV1_2
```

Comment (metacosm) [reply istio-model/src/main/java/me/snowdrop/istio/api/networking/v1beta1/ServerTLSSettingsProtocol.java:39]: Thanks, I've fixed the underlying issue and added a test.

**Pull request merged**